### PR TITLE
feat: add text props

### PR DESCRIPTION
**Pull request by mtnptrsn**
### Changes
- [x] add deco prop
- [x] add lowercase prop
- [x] add thin prop
- [x] remove family prop

### CODE REVIEW

File: src/lib/types/text.ts [hunk 1]
```diff
@@ -14,12 +14,13 @@ export interface ITextProps extends IGeneralProps, TextProps {
...
-  family?: string
   o?: number
   /**
    * Utlity props
    */
   uppercase?: boolean
   capitalize?: boolean
   bold?: boolean
+  thin?: boolean
+  deco?: 'underline' | 'none' | 'through' | 'underline-through'
```
Comment (kimastrom) [inline src/lib/types/text.ts]: These are invalid values for textDecorationLine

available values are: textDecorationLine: enum('none', 'underline', 'line-through', 'underline line-through')

https://facebook.github.io/react-native/docs/text#style

File: src/lib/utils/getTextChildren.ts [hunk 2]
```diff
@@ -4,6 +4,7 @@ const capitalize = (s: string) => {
 }
 
 const getTextChildren = (props: any, text: string) => {
+  if (props.lowercase) text = text.toLowerCase()
```
Comment (kimastrom) [inline src/lib/utils/getTextChildren.ts:7]: Typing are missing for this prop

**Review by kimastrom — CHANGES_REQUESTED** [review]

File: src/lib/types/text.ts (hunk 1 above)
Comment (mtnptrsn) [reply src/lib/types/text.ts]: This is now fixed!

File: src/lib/utils/getTextChildren.ts (hunk 2 above)
Comment (mtnptrsn) [reply src/lib/utils/getTextChildren.ts:7]: What do you mean ?

Comment (kimastrom) [reply src/lib/utils/getTextChildren.ts:7]: Lowercase is missing in Text component interface.

Comment (mtnptrsn) [reply src/lib/utils/getTextChildren.ts:7]: Aaaah

Comment (mtnptrsn) [reply src/lib/utils/getTextChildren.ts:7]: Fixed!

**Review by kimastrom — APPROVED** [review]

**Pull request merged**